hotel_calendar/static/src/js/views/calendar/hotel_calendar_controller.js

```diff
@@ -634,11 +635,23 @@ var PMSCalendarController = AbstractController.extend({
 
     _generate_reservation_tooltip_dict: function(tp) {
       return {
+        'folio_name': tp['folio_name'],
         'name': tp['name'],
         'phone': tp['phone'],
-        'arrival_hour': HotelCalendar.toMomentUTC(tp['checkin'], HotelConstants.ODOO_DATETIME_MOMENT_FORMAT).local().format('HH:mm'),
+        'email': tp['email'],
+        'room_type': tp['room_type'],
+        'adults': tp['adults'],
+        'children': tp['children'],
+        'checkin': HotelCalendar.toMomentUTC(tp['checkin'], '').format("DD MMMM"),
```

Comment:
Now toMomentUTC make no sense... best use moment.js directly: ```moment(tp['checkin'])```

HotelCalendar works with 'little-endian' date format... but if you manage odoo records, both, odoo and moment uses 'big-endian' date format as default.